src/components/Router/index.js

```diff
@@ -205,11 +204,11 @@ class HistoryRouter extends Component {
 
   initialStep = () => this.state.initialStep === this.state.step && this.state.flow === 'captureSteps'
 
-  changeFlowTo = (newFlow, newStep = 0, entriesBack = 1) => {
+  changeFlowTo = (newFlow, newStep = 0, addStepToHistory) => {
```

Comment:
which one is called outside of this component, `changeFlowTo` or `setStepIndex`. If it's `changeFlowTo` then the default value for `addStepToHistory` should be set there.